.env-devel

```diff
@@ -21,6 +21,7 @@ AGENT_VOLUMES_CLEANUP_S3_SECRET_KEY=12345678
 API_SERVER_DEV_FEATURES_ENABLED=0
 API_SERVER_LOGLEVEL=WARNING
 API_SERVER_PROFILING=1
+API_SERVER_INFLIGHTREQ_AMOUNT=25
```

Comment:
THOUGHT: I am afraid that at some point we might drop this env var if we do not see it in the api-server settings. I would add some reference to traefik in the name or prefix it as `TRAFIK_`

Reply:
Valid point. I will change.